src/components/panels/Temperature/TemperaturePanelList.vue

```diff
@@ -93,6 +112,9 @@ export default class TemperaturePanelList extends Mixins(BaseMixin) {
                 // hide MCU & Host sensors, if the function is enabled
                 if (this.hideMcuHostSensors && this.checkMcuHostSensor(fullName)) return false
 
+                // hide monitors, if the function is enabled
+                if (this.hideMonitors && this.available_monitors.includes(fullName)) return false
```

Comment:
remove unneeded filter condition.
We're filtering the available_sensors here. This will never be true because there is no available_monitors entry in this array.

Reply:
i removed the filter...